app/javascript/lib/vue-components/modules/Table.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div
+    data-testid="table-component"
+    class="gobierto-table"
+  >
+    <div class="gobierto-table__header">
+      <slot name="title" />
+      <slot
+        name="columns"
+        :toggle-visibility="toggleVisibility"
+      />
+    </div>
+    <table>
+      <thead>
+        <template v-for="[id, { name, index, cssClass }] in columnsMapArr">
+          <th
+            :key="index"
+            class="gobierto-table__th"
+            :class="cssClass"
+            @click="handleTableHeaderClick(id)"
+          >
+            {{ name }}
+            <SortIcon
+              v-if="currentSortColumn === id"
+              :direction="getSorting(id)"
+            />
+          </th>
+        </template>
+      </thead>
+      <tbody>
+        <template
+          v-for="(item, index) in dataTable"
+        >
+          <router-link
+            :key="index"
+            :to="{ name: routingMember, params: {id: item[routingId] } }"
+            tag="tr"
+            class="gobierto-table__tr"
+          >
+            <template
+              v-for="{key, field, type, cssClass} in filterColumns"
+            >
+              <template v-if="type === 'money'">
+                <td
+                  :key="key"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>
+                    {{ money(item[field]) }}
+                  </span>
+                </td>
+              </template>
+              <template v-else-if="type === 'date'">
+                <td
+                  :key="key"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>
+                    {{ item[field] }}
+                  </span>
+                </td>
+              </template>
+              <template v-else-if="type === 'link'">
+                <td
+                  :key="key"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>
+                    <a href="#">{{ item[field] }}</a>
+                  </span>
+                </td>
+              </template>
+              <template v-else-if="type === 'truncate'">
```

Comment:
What's a truncate cell type? Indeed, what's the difference with the default one? It's just wrapped with an `span`?